### PR TITLE
Bz minor fixes

### PR DESCRIPTION
**Pull request by maorlipchuk**

### CODE REVIEW

File: tasks/generate_mapping.py [hunk 1]
```diff
@@ -235,11 +235,13 @@ def _write_attached_storage_domains(f, dc_service, dc):
                 f.write("  dr_primary_address: %s\n" %
                         attached_sd.storage.volume_group
                         .logical_units[0].address)
-                targets = ''
+                targets = set()
```
Comment (machacekondra) [inline tasks/generate_mapping.py]: targets = set(lun_unit.target for lun_unit in attached_sd.storage.volume_group.logical_units)

File: tasks/recover_engine.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@
           username: "{{ vars['dr_sites_' + dr_target_host + '_username'] }}"
           password: "{{ vars['dr_sites_' + dr_target_host + '_password'] }}"
           ca_file: "{{ vars['dr_sites_' + dr_target_host + '_ca_file'] }}"
+      ignore_errors: False
```
Comment (machacekondra) [inline tasks/recover_engine.yml]: ignore_erros: false is default, so why do you need it?

Comment (maorlipchuk) [reply tasks/recover_engine.yml]: the entire block has ignore_errors: True (see line 158 in the file) so only the action related to the storage domains should be false all the time

File: tasks/generate_mapping.py (hunk 1 above)
Comment (maorlipchuk) [reply tasks/generate_mapping.py]: Thanks, will do

**Pull request merged**